pages/recommend.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback, useState } from 'react';
+import firebase from 'firebase';
+import 'firebase/database';
+import { useAuthUser, withAuthUser, AuthAction } from 'next-firebase-auth';
+import { useRouter } from 'next/dist/client/router';
+
+import Header from '../components/Header';
+import { RecommendForm } from '../src/pages/recommend/RecommendForm';
+
+if (!firebase.apps.length) {
+  firebase.initializeApp({
+    apiKey: process.env.NEXT_PUBLIC_FIREBASE_PUBLIC_API_KEY,
+    authDomain: process.env.NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN,
+    databaseURL: process.env.NEXT_PUBLIC_FIREBASE_DATABASE_URL,
+    projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
+  });
+}
+
+const Recommend = () => {
+  const AuthUser = useAuthUser();
+  const message = 'This is the page for adding recommendations!';
+  const [isLoading, setLoading] = useState(false);
+  const router = useRouter();
+
+  const handleFormSubmit = useCallback(
```

Comment:
👀 nice! Alternatively we can send the payload to the `/merchants` API route and save the data in the DB from there. I could also make a follow-up PR for this 🙂

Reply:
yes please! i think doesn't matter how, as it's kinda the same approach for FE :)